website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -98,6 +98,28 @@ components: sources: opentelemetry: {
 		}
 	}
 
+	configuration_examples: [
+		{
+			title: "OTLP Defaults"
```

Comment:
Is TLS the default? I could see it being unneeded on localhost.

Reply:
TLS is enabled by default on the otel-col: https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls--mtls-configuration

Reply:
Added in 18a095957